doc/api-extensions.md

```diff
@@ -1228,4 +1228,7 @@ This adds support for virtual GPUs. It introduces the `mdev` config key for GPU
 a supported mdev type, e.g. i915-GVTg_V5_4.
 
 ## resources\_pci\_iommu
-This adds the IOMMUGroup field for PCI entries in the resources API.
\ No newline at end of file
+This adds the IOMMUGroup field for PCI entries in the resources API.
+
+## instance\_nic\_port\_isolation
```

Comment:
Can you call this `instance\_nic\_bridged\_port\_isolation` as it only affects `bridged` NICs, not other types of NICs.

Also can you expand the description to clarify that, e.g.

"This introduces the `security.port_isolation` setting for `bridged` NICs."